examples/submeter_hierarchies/analyze.py

```diff
@@ -0,0 +1,78 @@
+import brickschema
+
+
+def get_graph(filename):
+    g = brickschema.Graph()
+    # load local Brick file
+    g.load_file("../../Brick.ttl")
+    g.load_file(filename)
+    g.expand("shacl")
+    return g
+
+
+FILES = [
+    "main-and-submeter.ttl",
+    "multiple-submeters.ttl",
+    "single-meter.ttl",
+]
+
+"""
+Gets all meters with an electrical power sensor which measure
+max power and returns their uuid
+"""
+all_meter_query = """SELECT DISTINCT * WHERE {
+    ?meter rdf:type/rdfs:subClassOf* brick:Meter .
+    ?meter brick:hasPoint ?point .
+    ?point rdf:type brick:Electric_Power_Sensor ;
+           brick:aggregate [ brick:aggregationFunction "max" ] ;
+           brick:timeseries [ brick:hasTimeseriesId ?uuid ] .
+}
+"""
+
+for file in FILES:
+    g = get_graph(file)
+    g.serialize("/tmp/test.ttl", format="turtle")
+    print(f"All Meter Query for {file}")
+    for row in g.query(all_meter_query):
+        print(row)
+    print("-" * 80)
+
+"""
+Gets the building meter and its peak (max) power sensor
+"""
+building_meter_query = """SELECT DISTINCT * WHERE {
+    ?meter rdf:type/rdfs:subClassOf* brick:Meter .
+    ?meter brick:isMeterOf/rdf:type/rdfs:subClassOf* brick:Building .
```

Comment:
do we use `brick:isMeterOf` instead of `meters`?

Reply:
fixed!